frontend/src/metabase/static-viz/components/LineAreaBarChart/utils/series.unit.spec.ts

```diff
@@ -1028,3 +1014,228 @@ describe("getSeriesWithLegends", () => {
     });
   });
 });
+
+describe("reorderSeries", () => {
```

Comment:
This starts the new test for the new util `reorderSeries` that will reorder combochart series respecting `graph.series_order`